src/otx/engine/engine.py

```diff
@@ -1107,7 +1108,16 @@ def _build_trainer(self, **kwargs) -> None:
                 self._cache.update(strategy="xpu_single")
                 # add plugin for Automatic Mixed Precision on XPU
                 if self._cache.args.get("precision", 32) == 16:
-                    self._cache.update(plugins=[MixedPrecisionXPUPlugin()])
+                    msg = "XPU doesn't support fp16 now, so bfp16 will be used instead."
```

Comment:
The feedback from IPEX is that BF16 is preferable for computer vision

Reply:
I think we can remove warning then

Reply:
I think it's better to tell user that it's changed to bfp16 because user can think that he uses float16 mixed precision. How do you think about it?

Reply:
I don't think we need a warning, it's better to handle the situation by properly documenting engine's precision parameter, and overall add a docs page describing the specific of training on XPU. The latter improvement should be done in a standalone PR though.

Reply:
I see. then I'll remove it.